spring-boot-docs/src/main/asciidoc/appendix-application-properties.adoc

```diff
@@ -336,7 +336,8 @@ content into your application; rather pick only the properties that you need.
 	spring.mvc.dispatch-options-request=false # Dispatch OPTIONS requests to the FrameworkServlet doService method.
 	spring.mvc.favicon.enabled=true # Enable resolution of favicon.ico.
 	spring.mvc.ignore-default-model-on-redirect=true # If the content of the "default" model should be ignored during redirect scenarios.
-	spring.mvc.locale= # Locale to use.
+	spring.mvc.locale= # Locale to use. Enable `FixedLocaleResolver` bean.
```

Comment:
There shouldn't be a difference between the documentation here and the one on the field. The one of the field may be more verbose but not the other way around.